cmd/minishift/cmd/config/config_test.go

```diff
@@ -50,11 +50,11 @@ var configTestCases = []configTestCase{
     "vm-driver": "kvm"
 }`,
 		config: map[string]interface{}{
-			"iso-url":   "http://foo.bar/minishift-centos.iso",
-			"vm-driver": "kvm",
-			"cpus":      4,
-			"disk-size": "20g",
-			"v":         5,
+			"iso-url":                   "http://foo.bar/minishift-centos.iso",
+			"vm-driver":                 "kvm",
+			"cpus":                      4,
+			"disk-size":                 "20g",
+			"v":                         5,
```

Comment:
:+1: , I am not sure why that was not giving the error in case of fmt when using go1.10.x

Reply:
that is a new thing... and I would even think a regression or a particular choice; the leader is a single space. maybe this is not distinct enough.